### ISSUE TITLE
App crashes after update via sync on Android

### ISSUE THREAD

**Issue by giladappsforce** [microsoft/react-native-code-push#2411]
ONLY RELEVANT FOR ANDROID, on Iphone it works as expected. 

### Steps to Reproduce
1. Open app after succesfull OTA update
2. wait for the update to finish (app is updating via the Codepush.sync method)
3. app crashes instead of restarting

### Expected Behavior
The app should restart.

### Actual Behavior
The app crashes after attempted restart.
```
01-04 16:22:44.808   724   923 I SDM     : HWInfo::GetFirstDisplayInterfaceType: First display is internal display
01-04 16:22:44.820  1989 29574 W ActivityManager:   Force finishing activity com.validit.stage/com.validit.MainActivity
01-04 16:22:44.823  1989 29574 D ActivityManager: setVisibility false ActivityRecord{ca47a0f u0 com.validit.stage/com.validit.MainActivity t621 f}
01-04 16:22:44.824  1989  2269 I BootReceiver: Copying /data/tombstones/tombstone_02 to DropBox (SYSTEM_TOMBSTONE)
01-04 16:22:44.826  1989  2179 I SystemUI[Framework]: ==> disableLocked() userId=0, what=0x0, token=android.os.Binder@80427b0, pkg=Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}, whichFlag=1, bar=com.android.internal.statusbar.IStatusBar$Stub$Proxy@b919e00
01-04 16:22:44.826  1989  2179 I SystemUI[Framework]: StatusBar disabled: mDisabled1=0x0 mDisabled2=0x0
01-04 16:22:44.826  1989  2179 I SystemUI[Framework]:   mDisableRecords.size=2
01-04 16:22:44.826  1989  2179 I SystemUI[Framework]:   [0] userId=0 what1=0x0 what2=0x0 pkg=null token=android.os.BinderProxy@d284d65
01-04 16:22:44.826  1989  2179 I SystemUI[Framework]:   [1] userId=0 what1=0x0 what2=0x0 pkg=null token=android.os.Binder@80427b0
01-04 16:22:44.837  1989  2178 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to com.google.android.gms/.stats.service.DropBoxEntryAddedReceiver
01-04 16:22:44.838  1989  2178 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to com.google.android.gms/.chimera.GmsIntentOperationService$PersistentTrustedReceiver
01-04 16:22:44.839  3776  7856 W ContextImpl: Calling a method in the system process without a qualified user: android.app.ContextImpl.sendBroadcast:1032 android.content.ContextWrapper.sendBroadcast:444 com.lge.mlt.providers.LDBSmartCareTrigger.sendMsgSmartCare:33 com.lge.mlt.providers.LDBMainLogProvider.insert:275 android.content.ContentProvider$Transport.insert:268 
01-04 16:22:44.844   816   816 E lowmemorykiller: Error writing /proc/29440/oom_score_adj; errno=22
01-04 16:22:44.848  1989  7476 W ActivityManager: Sending non-protected broadcast com.lge.mlt.action.smartcare.event from system 3776:com.lge.mlt/1000 pkg com.lge.mlt
01-04 16:22:44.851  1989  2178 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to com.google.android.gms/.stats.service.DropBoxEntryAddedReceiver
01-04 16:22:44.855  1989  2178 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to com.google.android.gms/.chimera.GmsIntentOperationService$PersistentTrustedReceiver
01-04 16:22:44.873  1989  2180 I ActivityManager: Start proc 29578:com.lge.ia.task.smartcare/1000 for broadcast com.lge.ia.task.smartcare/.SmartCareIntentReceiver
01-04 16:22:44.890 29578 29578 E .task.smartcar: Not starting debugger since process cannot load the jdwp agent.
01-04 16:22:44.903  1989  2580 W InputDispatcher: channel '2ebf42a com.validit.stage/com.validit.MainActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
01-04 16:22:44.903  1989  2580 E InputDispatcher: channel '2ebf42a com.validit.stage/com.validit.MainActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
01-04 16:22:44.906  1081  1081 I Zygote  : Process 29440 exited due to signal (6)
01-04 16:22:44.906  1989  2845 D ConnectivityService: ConnectivityService NetworkRequestInfo binderDied(NetworkRequest [ TRACK_DEFAULT id=1572, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Unwanted:  Uid: 10421] ], android.os.BinderProxy@297bed8)
01-04 16:22:44.906  1989  2845 D DisplayManagerService: Display listener for pid 29440 died.
01-04 16:22:44.907  1989  8413 I WindowManager: WIN DEATH: Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}
01-04 16:22:44.907  1989  8413 W InputDispatcher: Attempted to unregister already unregistered input channel '2ebf42a com.validit.stage/com.validit.MainActivity (server)'
01-04 16:22:44.907  1989  3682 I ActivityManager: Process com.validit.stage (pid 29440) has died: vis  +99TOP 
01-04 16:22:44.907  1989  2181 W libprocessgroup: kill(-29440, 9) failed: No such process
01-04 16:22:44.907  1989  2181 I libprocessgroup: Successfully killed process cgroup uid 10421 pid 29440 in 0ms
01-04 16:22:44.907  1989  2620 D ConnectivityService: releasing NetworkRequest [ TRACK_DEFAULT id=1572, [ Capabilities: INTERNET&NOT_RESTRICTED&TRUSTED Unwanted:  Uid: 10421] ] (release request)
01-04 16:22:44.911  1989  8413 D InputDispatcher: Focus left window: Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}
01-04 16:22:44.911  1989  8413 D InputDispatcher: Window went away: Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}
01-04 16:22:44.918   817   882 W SurfaceFlinger: Attempting to set client state on removed layer: com.validit.stage/com.validit.MainActivity@29440#1
01-04 16:22:44.918   817   882 W SurfaceFlinger: Attempting to destroy on removed layer: com.validit.stage/com.validit.MainActivity@29440#1
01-04 16:22:44.918   727   727 D QTI PowerHAL: notify_prm_interaction
01-04 16:22:44.918  1989  3682 D PowerManagerServiceEx: releaseWakeLockInternal: lock=204086057 [WindowManager], flags=0x0
01-04 16:22:44.922   817   817 I SurfaceFlinger: Removed ContainerLayer(0x72de3ac000) 2ebf42a com.validit.stage/com.validit.MainActivity#0 (5760 x 5760)
01-04 16:22:44.922   817   817 I SurfaceFlinger: Removed ContainerLayer(0x72e06ee000) AppWindowToken{3d2ca5 token=Token{5d2ba9c ActivityRecord{ca47a0f u0 com.validit.stage/com.validit.MainActivity t621}}}#0 (5760 x 5760)
01-04 16:22:44.924   817   817 I SurfaceFlinger: Removed BufferLayer(0x72e06fa000) com.validit.stage/com.validit.MainActivity@29440#1 (1440 x 2880)
01-04 16:22:44.925   724   923 D SDM     : DisplayBase::BuildLayerStackStats: LayerStack layer_count: 3, app_layer_count: 2, gpu_target_index: 2, display type: 0
01-04 16:22:44.926   724   923 I SDM     : HWInfo::GetFirstDisplayInterfaceType: First display is internal display
01-04 16:22:44.932  1989  3682 D ActivityManager: setVisibility true ActivityRecord{438c972 u0 com.lge.launcher3/.AllAppsLauncherExtension t369}
01-04 16:22:44.937   767   767 W PHHandler: [PH] hint not found.
01-04 16:22:44.937   767   767 W PHHandler: [PH] hint not found.
01-04 16:22:44.938  1076  1111 I PowerRM : System FW:1 IC:0 SO:0 GT:0 CAPP:-1 APP:0
01-04 16:22:44.938  1076  1111 I PowerRM : FIA lock1 locked
01-04 16:22:44.934  1989  3682 D ActivityManager: setVisibility true ActivityRecord{438c972 u0 com.lge.launcher3/.AllAppsLauncherExtension t369}
01-04 16:22:44.938  2812  2836 D AutoGameModeMonitor: Display : 0  Package : u0 com.lge.launcher3  PrevPackage : u0 com.validit.stage
01-04 16:22:44.938  2812  2831 D VideoActivityMonitorService: VE prop enable: false
01-04 16:22:44.939   817   817 I SurfaceFlinger: Removed ContainerLayer(0x72e0646000) Stack=296#0 (1440 x 2880)
01-04 16:22:44.939   817   817 I SurfaceFlinger: Removed BlurLayer(0x72e06c2000) animation background stackId=296#0 (5760 x 5760)
01-04 16:22:44.939   817   817 I SurfaceFlinger: Removed ContainerLayer(0x72e06d3000) Task=621#0 (5760 x 5760)
01-04 16:22:44.940   724   923 D SDM     : DisplayBase::BuildLayerStackStats: LayerStack layer_count: 3, app_layer_count: 2, gpu_target_index: 2, display type: 0
01-04 16:22:44.940   814  1443 V AudioFlinger: AudioFlinger::setRecordSilenced(uid:10421, silenced:1)
01-04 16:22:44.941   724   923 I SDM     : HWInfo::GetFirstDisplayInterfaceType: First display is internal display
01-04 16:22:44.949  1989  2272 W ActivityManager: setHasOverlayUi called on unknown pid: 29440
01-04 16:22:44.953  2683  2683 I GameSoundHandler: setSoundMode mDBStatus = false / status = false
01-04 16:22:44.960  1989  3681 D WindowManager: un - registerGameMode display ID 0
01-04 16:22:44.961  3945  3945 I [LGHome6]WallpaperMotionManager: isMotionEnabled: false
01-04 16:22:44.961  3945  3945 I [LGHome6]Launcher: mBindOnResumeCallbacks.size() : 0
01-04 16:22:44.962  1989  2179 I SystemUI[Framework]: ==> disableLocked() userId=0, what=0x0, token=android.os.Binder@80427b0, pkg=Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}, whichFlag=1, bar=com.android.internal.statusbar.IStatusBar$Stub$Proxy@b919e00
01-04 16:22:44.962  1989  2179 I SystemUI[Framework]: StatusBar disabled: mDisabled1=0x0 mDisabled2=0x0
01-04 16:22:44.962  1989  2179 I SystemUI[Framework]:   mDisableRecords.size=1
01-04 16:22:44.962  1989  2179 I SystemUI[Framework]:   [0] userId=0 what1=0x0 what2=0x0 pkg=null token=android.os.BinderProxy@d284d65
01-04 16:22:44.963  1989  2179 I SystemUI[Framework]: ==> disableLocked() userId=0, what=0x0, token=android.os.Binder@80427b0, pkg=Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}, whichFlag=1, bar=com.android.internal.statusbar.IStatusBar$Stub$Proxy@b919e00
01-04 16:22:44.963  1989  2179 I SystemUI[Framework]: StatusBar disabled: mDisabled1=0x0 mDisabled2=0x0
01-04 16:22:44.963  1989  2179 I SystemUI[Framework]:   mDisableRecords.size=2
01-04 16:22:44.963  1989  2179 I SystemUI[Framework]:   [0] userId=0 what1=0x0 what2=0x0 pkg=null token=android.os.BinderProxy@d284d65
01-04 16:22:44.963  1989  2179 I SystemUI[Framework]:   [1] userId=0 what1=0x0 what2=0x0 pkg=null token=android.os.Binder@80427b0
01-04 16:22:44.964  1989  2179 I SystemUI[Framework]: ==> disableLocked() userId=0, what=0x0, token=android.os.Binder@80427b0, pkg=Window{2ebf42a u0 com.validit.stage/com.validit.MainActivity}, whichFlag=1, bar=com.android.internal.statusbar.IStatusBar$Stub$Proxy@b919e00
01-04 16:22:44.964  1989  2179 I SystemUI[Framework]: StatusBar disabled: mDisabled1=0x0 mDisabled2=0x0
01-04 16:22:44.964  1989  2179 I SystemUI[Framework]:   mDisableRecords.size=1
01-04 16:22:44.964  1989  2179 I SystemUI[Framework]:   [0] userId=0 what1=0x0 what2=0x0 pkg=null token=android.os.BinderProxy@d284d65
01-04 16:22:44.972  3945  3945 I [LGHome6]StaticBlurEngine: getBlurImage 360 x 720 -> (0,0)
01-04 16:22:44.973   724   923 D SDM     : DisplayBase::BuildLayerStackStats: LayerStack layer_count: 3, app_layer_count: 2, gpu_target_index: 2, display type: 0
01-04 16:22:44.974   724   923 I SDM     : HWInfo::GetFirstDisplayInterfaceType: First display is internal display
01-04 16:22:44.974  3945  3945 I [LGHome6]HomescreenBlurManager: updateBackgroundViewContents() : mBackgroundViewImage(android.graphics.Bitmap@7790693)
```

### Environment

* react-native-code-push version: 7.0.5
* react-native version: 0.69.7
* iOS/Android/Windows version: Android version 9
* Does this reproduce on a debug build or release build? : Release
* Does this reproduce on a simulator, or only on a physical device : Physical device, can't test on simulator

tried: 
https://github.com/microsoft/react-native-code-push/issues/2052#issuecomment-837138801
https://github.com/microsoft/react-native-code-push/issues/1961#issuecomment-710698621
both didn't work.

https://user-images.githubusercontent.com/114646525/210578563-c6646e12-5b41-4961-b610-dab91e92645d.mp4

**Comment by pinvisible** [comment]
Yes, I am facing same issue with iOS app too!

**Comment by QuangBinhDinh** [comment]
Same here on android

**Comment by TacticCoder** [comment]
Same here on ios

**Comment by FiratYildiz-Lingua** [comment]
Hello, I'm having this issue on iOS myself, the app crashes immediately when the update is downloaded, and rolls back on the next launch.

Here's the crash log from Firebase Crashlytics:

```
Fatal Exception: RCTFatalException: Could not get BatchedBridge, make sure your bundle is packaged correctly
0  CoreFoundation                 0x1217e0 (Missing UUID b02805dc7bce35718eb6ff671506488b)
1  libobjc.A.dylib                0x25144 (Missing UUID ebfccc6696d434d386b4a0b3937a18ac)
2  <appName>                         0x28dd58 RCTFormatError + 167 (RCTAssert.m:167)
3  <appName>                         0x2a315c -[RCTCxxBridge handleError:] + 1180 (RCTCxxBridge.mm:1180)
4  <appName>                         0x2a1a58 __34-[RCTCxxBridge _initializeBridge:]_block_invoke + 688 (RCTCxxBridge.mm:688)
5  <appName>                         0x2b4c34 facebook::react::RCTMessageThread::tryFunc(std::__1::function<void ()> const&) + 73 (RCTMessageThread.mm:73)
6  <appName>                         0x2b49d0 invocation function for block in facebook::react::RCTMessageThread::runAsync(std::__1::function<void ()>) + 46 (RCTMessageThread.mm:46)
7  CoreFoundation                 0x85fa4 (Missing UUID b02805dc7bce35718eb6ff671506488b)
8  CoreFoundation                 0x856ec (Missing UUID b02805dc7bce35718eb6ff671506488b)
9  CoreFoundation                 0x7fef0 (Missing UUID b02805dc7bce35718eb6ff671506488b)
10 CoreFoundation                 0x7f7f4 (Missing UUID b02805dc7bce35718eb6ff671506488b)
11 <appName>                         0x29f5e8 +[RCTCxxBridge runRunLoop] + 335 (RCTCxxBridge.mm:335)
12 Foundation                     0x4e4aac (Missing UUID 532aae62f6cf3de28988170389a83ad5)
13 libsystem_pthread.dylib        0x74e4 (Missing UUID 030d2641514f373f9688582d9d3b369a)
14 libsystem_pthread.dylib        0x26cc (Missing UUID 030d2641514f373f9688582d9d3b369a)
```

# Environment
- react-native-code-push version: 7.0.4
- react-native version: 0.70.6
- appcenter-cli version: 2.12.0
- iOS/Android/Windows version: iOS 16.1
- Does this reproduce on a debug build or release build? release
- Does this reproduce on a simulator, or only on a physical device? simulator

**Comment by ko-devHong** [comment]
maybe this issue fix in new version `7.1.0`

ref : https://github.com/microsoft/react-native-code-push/pull/2337

**Comment by iita71737** [comment]
Same here on android

 "react-native-code-push": "8.0.2",
  "react-native": "0.72.4",
